File: labServiceUIJava/labServiceUIJava/src/main/java/com/example/labserviceuijava/service/Service.java
package com.example.labserviceuijava.service;

public class Service {

    private final ServiceArtist serviceArtist;

    private final ServiceCashier serviceCashier;

    private final ServiceConcert serviceConcert;

    private final ServiceOrder serviceOrder;

    public Service(ServiceArtist serviceArtist, ServiceCashier serviceCashier, ServiceConcert serviceConcert, ServiceOrder serviceOrder) {
        this.serviceArtist = serviceArtist;
        this.serviceCashier = serviceCashier;
        this.serviceConcert = serviceConcert;
        this.serviceOrder = serviceOrder;
    }

    public ServiceArtist getServiceArtist() {
        return serviceArtist;
    }

    public ServiceCashier getServiceCashier() {
        return serviceCashier;
    }

    public ServiceConcert getServiceConcert() {
        return serviceConcert;
    }

    public ServiceOrder getServiceOrder() {
        return serviceOrder;
    }
}

File: labServiceUIcs/WindowsFormsApp2/service/Service.cs
namespace WindowsFormsApp2
{
    public class Service
    {
        private ServiceArtist serviceArtist;

        private ServiceCashier serviceCashier;

        private ServiceConcert serviceConcert;

        private ServiceOrder serviceOrder;

        public Service(ServiceArtist serviceArtist, ServiceCashier serviceCashier, ServiceConcert serviceConcert,
            ServiceOrder serviceOrder)
        {
            this.serviceArtist = serviceArtist;
            this.serviceCashier = serviceCashier;
            this.serviceConcert = serviceConcert;
            this.serviceOrder = serviceOrder;
        }

        public ServiceArtist ServiceArtist
        {
            get { return serviceArtist; }
        }

        public ServiceCashier ServiceCashier
        {
            get { return serviceCashier; }
        }

        public ServiceConcert ServiceConcert
        {
            get { return serviceConcert; }
        }

        public ServiceOrder ServiceOrder
        {
            get { return serviceOrder; }
        }
    }
}